internal/cli/cmd/sysdump.go

```diff
@@ -74,6 +74,9 @@ func initSysdumpFlags(cmd *cobra.Command, options *sysdump.Options, optionPrefix
 	cmd.Flags().StringVar(&options.CiliumEnvoyLabelSelector,
 		optionPrefix+"cilium-envoy-label-selector", sysdump.DefaultCiliumEnvoyLabelSelector,
 		"The labels used to target Cilium Envoy pods")
+	cmd.Flags().StringVar(&options.CiliumHelmChartReleaseName,
+		"cilium-helm-chart-release-name", sysdump.DefaultCiliumHelmChartReleaseName,
+		"Cilium Helm chart release name to get the values from the sysdump")
```

Comment:
This description is a bit redundant in context. A better one would be 

"The release name to use when retrieving Helm values"

Also, the word "chart" is a bit redundant here as well, as all releases are associated with a chart. So, this could be changed to `cilium-helm-release-name` and `CiliumHelmReleaseName`.

Reply:
will change in my next commit